packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -129,6 +146,8 @@ export class DeltaScheduler {
             this.schedulingCount++;
```

Comment:
Observation RE this.isScheduling & this.schedulingCount:

It feels to me that this.schedulingLog can be removed, as this.schedulingLog !== undefined can be used for same check. Make it a bit easier to follow code.

Also we do not count or report cases where it took us single batch to process all ops till idle. This skews data toward 2+ batches till idle. I think we should populate this.schedulingLog in batchEnd() even if queue is empty at that moment.

Reply:
That should do it. I also moved the initialization of the schedulingLog to happen before the check to see if the queue is empty (shouldRunScheduler)